rest/route/notification.go

```diff
@@ -0,0 +1,331 @@
+func (h *jiraCommentNotificationPostHandler) Run(ctx context.Context) gimlet.Responder {
+	i, err := h.APIJiraComment.ToService()
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "API error converting from model.APIJiraComment to message.JIRAComment"))
+	}
+	comment, ok := i.(*message.JIRAComment)
+	if !ok {
+		return gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
+			StatusCode: http.StatusInternalServerError,
+			Message:    fmt.Sprintf("Unexpected type %T for message.JIRAComment", i),
+		})
+	}
+
+	h.composer = message.MakeJIRACommentMessage(comment.IssueID, comment.Body)
+	h.sender, err = h.environment.GetSender(evergreen.SenderJIRAComment)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "Error fetching sender key for evergreen.SenderJIRAComment"))
+	}
+
+	h.sender.Send(h.composer)
+
+	return gimlet.NewJSONResponse(struct{}{})
+}
+
+///////////////////////////////////////////////////////////////////////
+//
+// POST /rest/v2/notifications/jira_issue
+
+type jiraIssueNotificationPostHandler struct {
+	APIJiraIssue *model.APIJiraIssue
+	composer     message.Composer
+	sender       send.Sender
+	environment  evergreen.Environment
+}
+
+func makeJiraIssueNotification(environment evergreen.Environment) gimlet.RouteHandler {
+	return &jiraIssueNotificationPostHandler{
+		environment: environment,
+	}
+}
+
+func (h *jiraIssueNotificationPostHandler) Factory() gimlet.RouteHandler {
+	return &jiraIssueNotificationPostHandler{
+		environment: h.environment,
+	}
+}
+
+// Parse fetches the JSON payload from the and unmarshals it to an APIJiraIssue.
+func (h *jiraIssueNotificationPostHandler) Parse(ctx context.Context, r *http.Request) error {
+	body := util.NewRequestReader(r)
+	defer body.Close()
+	b, err := ioutil.ReadAll(body)
+	if err != nil {
+		return errors.Wrap(err, "Argument read error")
+	}
+
+	h.APIJiraIssue = &model.APIJiraIssue{}
+	if err := json.Unmarshal(b, h.APIJiraIssue); err != nil {
+		errors.Wrap(err, "API error while unmarshalling JSON to model.APIJiraIssue")
+	}
+
+	return nil
+}
+
+// Run dispatches the notification.
+func (h *jiraIssueNotificationPostHandler) Run(ctx context.Context) gimlet.Responder {
+	i, err := h.APIJiraIssue.ToService()
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "API error converting from model.APIJiraIssue to message.JiraIssue"))
+	}
+	issue, ok := i.(*message.JiraIssue)
+	if !ok {
+		return gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
+			StatusCode: http.StatusInternalServerError,
+			Message:    fmt.Sprintf("Unexpected type %T for message.JiraIssue", i),
+		})
+	}
+
+	h.composer = message.MakeJiraMessage(issue)
+	h.sender, err = h.environment.GetSender(evergreen.SenderJIRAIssue)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "Error fetching sender key for evergreen.SenderJIRAIssue"))
+	}
+
+	h.sender.Send(h.composer)
```

Comment:
While this works I expected these routes to insert a notification document that would be picked up by the event_send amboy job. That way the behavior is consistent with what the app does and we can track them